setup.py

```diff
@@ -56,7 +56,7 @@ def package_files(directory):
         "xlrd",
         "stea",
         "pyscal>=0.4.0",
-        "fmu-ensemble",
+        "fmu-ensemble<=1.2.2",
```

Comment:
Think you need to pin it < 1.2.2 or maybe better with != 1.2.2, think that should work?

Reply:
ah, i thought you ment i should use 1.2.2... will try <1.2.2